presentation-api/src/main/java/org/cytoscape/view/presentation/property/BasicVisualLexicon.java

```diff
@@ -3,30 +3,19 @@
 import java.awt.Color;
```

Comment:
The code that was removed from BasicVisualLexicon was moved to AbstractVisualLexicon so it could be reused by BasicTableVisualLexicon.